.github/workflows/install.yml

```diff
@@ -11,6 +11,36 @@ defaults:
     shell: bash
 
 jobs:
+  debian:
```

Comment:
Missing Arch using https://hub.docker.com/_/archlinux image as the container tag

Reply:
Fixed. CI runs succesfully here: https://github.com/r0gue-io/pop-cli/actions/runs/9029075945